pkg/extensions/sync/content.go

```diff
@@ -63,6 +63,13 @@ func (cm ContentManager) FilterTags(repo string, tags []string) ([]string, error
 			}
 		}
 
+		if content.Tags.ExcludeRegex != nil {
+			tags, err = excludeTagsByRegex(tags, *content.Tags.ExcludeRegex, cm.log)
+			if err != nil {
+				return []string{}, err
+			}
```

Comment:
Can we add logs here, so if something funny happens, we figure it out easily?

Reply:
Shouldn't i in that case also log regex matched tags?

Or even more explicit - log what tags we have on the source, what selected by regex, what excluded and the resulting set?

Reply:
At the same time - i'm not sure that this really needed. Imagine report for something like `library/mariadb`, which have hundreds if tags (if not thousands already)...

Reply:
Maybe log those as debug messages. In any case we are logging the final result of the filtering in the `pkg/extensions/sync/service.go` file in function `SyncRepo`.